topdown/topdown_test.go

```diff
@@ -95,6 +95,50 @@ func TestTopDownQueryIDsUnique(t *testing.T) {
 	}
 }
 
+func TestTopDownIndexExpr(t *testing.T) {
+	ctx := context.Background()
+	store := inmem.New()
+	inputTerm := &ast.Term{}
```

Comment:
I'm surprised this is working! It should not be possible for a term.Value to be nil. It would be better to just leave this out entirely.

Reply:
Removed